receiver/databricksreceiver/README.md

```diff
@@ -0,0 +1,53 @@
+# Databricks Receiver
+
+The Databricks Receiver uses the Databricks
+[API](https://docs.databricks.com/dev-tools/api/latest/index.html)
+to generate metrics about the operation of a Databricks instance.
+
+In addition to generating metrics from the Databricks API, it also generates metrics from the Spark subsystem running in a Databricks instance.
+
+Supported pipeline types: `metrics`
+
+> :construction: This receiver is in **DEVELOPMENT**. Behavior, configuration fields, and metric data model are subject to change.
```

Comment:
It'd be nice to start with the metadata.yaml file to generate the header. Please check other components, e.g. https://github.com/open-telemetry/opentelemetry-collector-contrib/commit/74836e91bd0490fa489a33b98123806cbb34ae8a

Reply:
Ok, sure. Added.

Do we want to require that the skeleton PR have a `metadata.yaml` and generated code? If so should we include this guidance in the [contributing guidelines](https://github.com/open-telemetry/opentelemetry-collector/blob/main/CONTRIBUTING.md#when-adding-a-new-component)?

Reply:
I'll open a separate issue to discuss this. Thanks for calling it out.

Reply:
I meant just to define the `status` part of the metadata.yaml instead of writing warnings like this manually. 

Now, we can keep the metrics as well